src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const dailyPayment = 40;
+  const discountTwenty = 20;
+  const discountfifty = 50;
+
+  if (days < 3) {
+    result = days * dailyPayment;
+  }
+
+  if (days < 7 && days > 2) {
+    result = (days * dailyPayment) - discountTwenty;
+  }
+
+  if (days >= 7) {
+    result = (days * dailyPayment) - discountfifty;
+  }
```

Comment:
Do you really need to save it in result variable? Can't you just return the value immediately?
You can check for bigDiscount, then for basicDiscount and if none of checks was passed, then just return default price